src/utils/trust/who.Is.Involved.mapper.ts

```diff
@@ -0,0 +1,18 @@
+import * as Trust from 'model/trust.model';
+import { TrustWhoIsInvolved } from 'model/trust.page.model';
+
+const mapTrustWhoIsInvolvedToPage = (
```

Comment:
the other utils/trust/mapper modules have their own unit test, would you be able to add one for this too.

Reply:
Good point. Added the unit test